src/Microsoft.Performance.SDK/Processing/RequiresCompositeCookerAttribute.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Performance.SDK.Extensibility;
+using System;
+
+namespace Microsoft.Performance.SDK.Processing
+{
+    /// <summary>
+    /// This attribute is used on a class with static properties that return <see cref="TableDescriptor"/>
+    /// or on individual <see cref="TableDescriptor"/> static properties to indicate that a 
+    /// table described by the <see cref="TableDescriptor"/> requires the identified composite data cooker.
+    /// </summary>
+    [AttributeUsage(System.AttributeTargets.Class, AllowMultiple = true, Inherited = false)]
+    public sealed class RequiresCompositeCookerAttribute
+        : RequiresCookerAttribute
```

Comment:
Do we still need to inherit this? Could we update the code that finds these to go ahead and look for the new ones, and fall back to the old one if it doesn't find them?

Reply:
Yes but then we're changing our cooker discovery code and will have to remove the legacy finding path once we do RC1. Doing it this way mean it "just works"

Reply:
Okay sounds good.